ui/src/components/home/EventsSection.js

```diff
@@ -10,47 +9,112 @@ import {
   Link,
   Button,
   LinkOverlay,
-} from "@chakra-ui/react";
-import Slider from "react-slick";
-import "slick-carousel/slick/slick.css";
-import "slick-carousel/slick/slick-theme.css";
-import axios from "axios";
-import { DateTime } from "luxon";
-import { useEffect, useState } from "react";
-import { getRenderDate } from "../helpers/eventsHelpers";
-import EventsSectionCard from "./EventsSectionCards";
-import FeaturedEvent from "./FeaturedEvent";
+} from '@chakra-ui/react';
+import Slider from 'react-slick';
+import 'slick-carousel/slick/slick.css';
+import 'slick-carousel/slick/slick-theme.css';
+import axios from 'axios';
+import { DateTime } from 'luxon';
+import { useEffect, useRef, useState } from 'react';
+import { getRenderDate } from '../helpers/eventsHelpers';
+import EventsSectionCard from './EventsSectionCards';
 
-const allEventsText = "all events >";
+const allEventsText = 'See All Events';
+
+function SampleNextArrow(props) {
+  const { onClick } = props;
+  return (
+    <div
+      style={{
+        display: 'block',
+        right: '0%',
+        position: 'absolute',
+        top: '40%',
+        zIndex: 8,
+        cursor: 'pointer',
+      }}
+      onClick={onClick}
+    >
+      <img
+        src={process.env.PUBLIC_URL + 'images/home/NextArrow.png'}
+        width="50%"
+        alt="Arrow"
+      />
+    </div>
+  );
+}
+
+function SamplePrevArrow(props) {
+  const { onClick } = props;
+  return (
+    <div
+      style={{
+        display: 'block',
+        left: '3%',
+        position: 'absolute',
+        top: '40%',
+        zIndex: 8,
+        cursor: 'pointer',
+      }}
+      onClick={onClick}
+    >
+      <img
+        src={process.env.PUBLIC_URL + 'images/home/PrevArrow.png'}
+        width="50%"
+        alt="Arrow"
+      />
+    </div>
+  );
+}
 
 const EventsSection = () => {
   const [events, setEvents] = useState([]);
-
+  const [computedMargin, setComputedMargin] = useState();
+  const marginRef = useRef(null);
   const sliderSettings = {
     adaptiveHeight: true,
-    arrows: false,
-    centerMode: true,
+    centerMode: false,
     dots: false,
     focusOnSelect: true,
     infinite: false,
     slidesPerRow: 1,
     speed: 500,
     swipeToSlide: true,
     variableWidth: true,
+    slidesToShow: 1,
+    slidesToScroll: 1,
+    nextArrow: <SampleNextArrow />,
+    prevArrow: <SamplePrevArrow />,
```

Comment:
<img width="661" alt="image" src="https://user-images.githubusercontent.com/24607458/184646138-e5476a70-ccf0-4392-934c-bf534066820e.png">
please maket the arrow buttons larger.

[nitpick] it should also conditionally not render the left arrow if it's on the first event item and not render the right arrow if its on the last item. but i understand if this is difficult to implement

<img width="661" alt="image" src="https://user-images.githubusercontent.com/24607458/184646444-574a9346-6b1e-4a43-bde0-8b6f26190dd3.png">